pkgs/development/python-modules/price-parser/default.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytestCheckHook,
+  setuptools,
+  pytest-cov,
+  attrs,
+}:
+
+buildPythonPackage rec {
+  pname = "price-parser";
+  version = "0.4.0";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "scrapinghub";
+    repo = "price-parser";
+    tag = version;
+    hash = "sha256-9f/+Yw94SVvg9fl9zYR9YEMwAgKHwySG5cysPMomnA0=";
+  };
+
+  dependencies = [ attrs ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    pytest-cov
```

Comment:
We do not care about coverage and this should be pytest-cov-stub to not slow us down in any way